client/src/components/flight-details-by-direction.component.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import Button from "@material-ui/core/Button";
+import { Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import FlightTakeoffIcon from "@material-ui/icons/FlightTakeoff";
+import FlightLandIcon from "@material-ui/icons/FlightLand";
+import CustomTimeline from "../components/custom-timeline.component";
+
+const FlightDetailsByDirection = (props) => {
+  const useStyles = makeStyles({
+    detail: {
+      padding: "15px",
+      borderBottom: "1px solid #D3D3D3",
+    },
+    icon: {
+      fontSize: "40px",
+      paddingRight: "15px",
+    },
+    button: {
+      backgroundColor: "#ffa500",
+      "&:hover": {
+        backgroundColor: "#ff8c00",
```

Comment:
I like that you thought about the hover. You could also probably add the colors to the theme so we can keep the styling consistent. Nice work on the timeline!